ru/turing/courses/lesson5/Main.java

```diff
@@ -1,7 +1,184 @@
 package ru.turing.courses.lesson5;
 
+
 public class Main {
     public static void main(String[] args) {
         //todo вставить сюда сортировки пузырьком, сделать для каждой свой метод
+        int[] integer_mass = new int[]{8, 4, 9, 15, 26, -1, 0};
```

Comment:
```suggestion
        int[] integerMass = new int[]{8, 4, 9, 15, 26, -1, 0};
```
Используем CamelCase)
Исправь здесь и дальше